packages/react/src/Blankslate/Blankslate.tsx

```diff
@@ -128,7 +130,7 @@ function Blankslate({border, children, narrow, spacious}: BlankslateProps) {
         This is a workaround so we can use `@container` without upgrading `styled-components` to 6.x
         See [this comment](https://github.com/primer/react/pull/3869#discussion_r1392523030) for more info
       */}
-      <style type="text/css">{BlankslateContainerQuery}</style>
+      <style type="text/css" dangerouslySetInnerHTML={{__html: BlankslateContainerQuery}} />
```

Comment:
This is very smart 🔥 

I also think it is obscure to cause hydration mismatch when it is passed as children 😄